### PR TITLE
feat: path defined

### PR DESCRIPTION
**Pull request by AshutoshM29**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -281,6 +281,11 @@ iframe {
   justify-content: flex-start;
 }
 
+.badge-count .bg-away {
+  position: relative;
+  z-index: 99999999999;
```
Comment (rajaranjanmr) [inline styles.css:286]: this much z-index value is not good practice to give. You should not give this much high value

File: Pages/forgot.html [hunk 2]
```diff
@@ -110,7 +110,7 @@ <h2 class="form-title">Forget Password</h2>
             </form>
         </section>
         <div class="hero-login">
-            <img src="/Assets/Gifs/Reset password.gif password.gif" alt="Reset password">
+            <img src="/Assets/Gifs/Reset password.gif" alt="Reset password">
         </div>
```
Comment (rajaranjanmr) [inline Pages/forgot.html:115]: if intentionally you have left space then its fine but in good practice its not doable.
suggestion:remove space

**Review by rajaranjanmr — APPROVED** [review]

**Comment by AshutoshM29** [comment]
I'm closing this PR : reason wrong PR raised

**Comment by AshutoshM29** [comment]
I have closed this PR, reason wrong PR raised

**Pull request closed without merging**